public/.env.example.js

```
@@ -17,3 +17,4 @@ var WS_DEBOUNCE_PERIOD_MS = 100 // milliseconds, remove or set to 0 to disable d
 var WS_PING_PERIOD_MS = 30000
 var WS_PING_TIMEOUT_PERIOD_MS = 35000
 // var SENTRY_DSN = "https://...@sentry.io/..."; // optional, omit to disable Sentry
+// var HIDE_RESOURCES = True; // optional
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The // var HIDE_RESOURCES  key has incorrect delimiter

(IncorrectDelimiter)